QUARKUS-278.md

```diff
@@ -0,0 +1,54 @@
+# QUARKUS-278 - Test and verify Integration with RH-SSO 7.4
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-278
+
+Coverage should ensure the integration with RH-SSO 7.4 and Quarkus security features.
+For native mode similar test coverage as for JVM mode needs to be ensured.
+
+## Scope of the testing
+Following actions were taken to ensure test coverage and automation for native image:
+ - Define the scope of the testing
+ - Analyze the possible impact on Quarkus itself - e.g performance, compatibility
+ - Determine impact on test suites - e.g. new test suite, increased complexity of existing test suite, maintenance burden
+ - Determine impact on QE infrastructure - e.g. increased complexity of existing automation, maintenance burden, need for new external resources
+ - Check a need for involvement of other product/component teams
+
+### Impact on testsuites and testing automation:
+ - OpenShift TS framework needs to be enhanced to install RH-SSO 7.4 and/or other versions
+ - New test coverage for security scenarios needs to be added into OpenShift TS:
+   - Protect Services Applications with Roles
+   - Protect Services Applications with Roles using permissions managed by RH SSO
+   - Protect Web Applications with Roles
+   - Protect Multi Tenant applications
+   - Protect Service Applications using JWT
+   - Cover Expiration of Bearer Token for Services Applications
+   - User Revocation for Services Applications
```

Comment:
@rsvoboda these are all the scenarios I've covered so far, but some of them are not even merged into master yet. Therefore, pending to confirm which scenarios will be part of 1.7 coverage.

Reply:
Please mention also existing tests in OpenShift TS. 

And add a note similar to this text:
Quarkus 1.7 focused test development targets reasonable test coverage baseline.
Future test development targets Quarkus 1.11 to gain iteratively stronger confidence.

Reply:
The existing tests are the first two scenarios. I will mention it.

Reply:
Changed